src/components/Tabs/Tabs.scss

```diff
@@ -87,11 +87,11 @@ $focus-state-box-shadow-color: rgba(92, 106, 196, 0.8);
     // stylelint-disable-next-line selector-max-class
     .Title {
       border-bottom: border-width(thicker) solid color('indigo');
-
-      @media (-ms-high-contrast: active) {
-        background: ms-high-contrast-color('selected-text-background');
-        border-bottom-color: ms-high-contrast-color('selected-text-background');
-      }
+    }
+    @media (-ms-high-contrast: active) {
+      outline: 1px solid ms-high-contrast-color('selected-text');
+      background: ms-high-contrast-color('selected-text-background');
+      border-bottom-color: ms-high-contrast-color('selected-text-background');
```

Comment:
I don't believe tabs have been converted yet, so we might have to give this a second pass.

Reply:
I saw that, which was why I tagged you on this PR. This just adds high contrast mode changes so I can check them off in the rollout doc: https://docs.google.com/spreadsheets/d/1hUXmywBByL1124Nsn6OrORfgI1H2R7FX5DWliYQZW_I/edit#gid=796134139

Reply:
Ah ok, I was thinking that something might get lost after the conversion